app/Actions/Site/UpdateEnv.php

```
@@ -39,59 +39,125 @@ public function update(Site $site, array $input): void
             ]);
         }
 
-        $variables = $this->processVariables($site, $input);
-
-        $envContent = EnvParser::stringify($variables);
+        $variables = $this->resolveVariables($site, $input, $path);
 
         $site->server->os()->write(
             $path,
-            $envContent,
+            EnvParser::stringify($variables),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Helpers\EnvParser' in method 'update'. (undefined)

(StaticAccess)